packages/lesswrong/components/posts/PostsPage/PostsPage.jsx

```diff
@@ -210,6 +210,16 @@ if (Meteor.isServer) {
   URLClass = require('url').URL
 }
 
+function getProtocol(url) {
+  if (URLClass)
```

Comment:
Make it clearer that this branch always executes on the server, and the rest never executes on the server